test/JDBC/expected/charindex_replace_patindex-vu-verify.out

```diff
@@ -0,0 +1,271 @@
+
+EXEC babel_5144_p @src = 'abcḍèĎÈdedEDEabcd', @from = 'de', @to = '##'
+GO
+~~START~~
+varchar
+running replace
+~~END~~
+
+~~START~~
+nvarchar
+abcḍèĎÈ######abcd
+~~END~~
+
+~~START~~
+varchar
+running charindex
+~~END~~
+
+~~START~~
+int
+8
+~~END~~
+
+~~START~~
+varchar
+running patindex
+~~END~~
+
+~~START~~
+bigint
+8
+~~END~~
+
+
+SELECT babel_5144_f1('abcḍèĎÈdedEDEabcd', 'de', '##')
+SELECT babel_5144_f2('abcḍèĎÈdedEDEabcd', 'de')
+SELECT babel_5144_f3('abcḍèĎÈdedEDEabcd')
+GO
+~~START~~
+nvarchar
+abcḍèĎÈ######abcd
+~~END~~
+
+~~START~~
+nvarchar
+8
+~~END~~
+
+~~START~~
+nvarchar
+8
+~~END~~
+
+
+INSERT INTO babel_5144_t1 VALUES ('abcḍèĎÈdedEDEabcd', 'de', '##')
+INSERT INTO babel_5144_t2 VALUES ('abcḍèĎÈdedEDEabcd', 'de', '##')
+INSERT INTO babel_5144_t3(src, substr1, substr2) VALUES ('abcḍèĎÈdedEDEabcd', 'de', '##')
+GO
+~~START~~
+varchar
+========== trigger start ==========
+~~END~~
+
+~~START~~
+nvarchar#!#int#!#bigint
+abcḍèĎÈ######abcd#!#8#!#8
+abcḍèĎÈ######abcd#!#8#!#8
+~~END~~
+
+~~START~~
+varchar
+========== trigger end   ==========
+~~END~~
+
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+
+-- validate check constraint
+INSERT INTO babel_5144_t2 VALUES ('abcdabcd', 'de', 'de')
+GO
+~~ERROR (Code: 547)~~
+
+~~ERROR (Message: new row for relation "babel_5144_t2" violates check constraint "babel_5144_t2_check")~~
+
+INSERT INTO babel_5144_t2 VALUES ('deĎÈdedEDEabcd', 'de', '##')
+GO
+~~ERROR (Code: 547)~~
+
+~~ERROR (Message: new row for relation "babel_5144_t2" violates check constraint "babel_5144_t2_check1")~~
+
+INSERT INTO babel_5144_t2 VALUES ('aaaaaabcd', 'ab', '##')
+GO
+~~ERROR (Code: 547)~~
+
+~~ERROR (Message: new row for relation "babel_5144_t2" violates check constraint "babel_5144_t2_src_check")~~
+
+
+SELECT * FROM babel_5144_t2
```

Comment:
For this and all other SELECT statements, let's add ORDER BY clause to make the result consistent.

Reply:
All rows are the same, which is why I did not add order by clause.